### DISCUSSION TITLE
Hide "This website runs on Solara" notice

### DISCUSSION THREAD

**Discussion by maercu** [widgetti/solara#673] [Q&A] [answered]
Hi, is there an easy way to hide the "This website runs on Solara" notice at the bottom of websites?

**Comment by iisakkirotko** [comment, marked as answer]
Hey @maercu!

There isn't a proper way to do this as of right now. It's on our roadmap, but will probably take a while until we get around to it. In the meantime you can use the workaround from [here](https://www.answeroverflow.com/m/1177169439855235072)

**Reply by maercu** [reply]
Thanks @iisakkirotko

**Comment by ntjess** [comment]
I thought you can simply run `solara run --theme-loader plain /path/to/file.py`

It works for me

(You can also set an environment variable: `SOLARA_THEME_LOADER=plain`

**Reply by Kugeleis** [reply]
This should be selected as the answer.

**Comment by maartenbreddels** [comment]
See https://github.com/widgetti/solara/pull/836

Use:
```
SOLARA_THEME_SHOW_BANNER=False solara run myapp.py
```
to get rid of the banner
